bigchaindb/tendermint/lib.py

```diff
@@ -191,6 +191,22 @@ def get_block(self, block_id, include_status=False):
         else:
             return block
 
+    def get_blocks_status_containing_tx(self, txid):
+        """Retrieve the list of blocks (block ids) containing a
+           transaction with transaction id `txid`
+
+        Args:
+            txid (str): transaction id of the transaction to query
+
+        Returns:
+            Block id list (list(int))
+        """
+        blocks = list(backend.query.get_block_with_transaction(self.connection, txid))
+        if len(blocks) > 1:
```

Comment:
I wonder if we really need this check. This is Tendermint's responsability.

Reply:
I agree this is Tendermint's responsibility but aim here to generate a meaningful log message in case state is critically inconsistant.